test/test-comment4.fire

```diff
@@ -0,0 +1 @@
+//
```

Comment:
we're not support inline comments, only block comments

Reply:
deleted